testsuites/syncgateway/functional/tests/test_xattrs.py

```diff
@@ -632,7 +632,7 @@ def test_large_initial_import(params_from_base_test_setup, sg_conf_name):
         connection_url = "couchbase://{}/{}?ipv6=allow".format(cbs_ip, bucket_name)
     else:
         connection_url = 'couchbase://{}/{}'.format(cbs_ip, bucket_name)
-    sdk_client = Bucket(connection_url, password='password')
+    sdk_client = Bucket(connection_url, password='password', timeout=30)
```

Comment:
is default timeout is not sufficient here?

Reply:
default is 5 second and on slower machine it fails. Anyways providing an higher timeout doesn't affect the runtime. When we are inserting bulk docs (around 30k) they are gonna take some time >5 and < 30. Any operation won't wait unnecessary for 30sec.